lib/client/ipam_block_reader_writer.go

```diff
@@ -92,6 +93,17 @@ func (rw blockReaderWriter) claimNewAffineBlock(
 		}
 	}
 
+reqPools:
```

Comment:
Good point :) but I wonder if creating a map (memory) and doing `.String()` (cpu) costs us more than just n^2 for a small n